invenio_users_resources/services/permissions.py

```diff
@@ -21,11 +22,11 @@
 class UsersPermissionPolicy(BasePermissionPolicy):
     """Permission policy for users and user groups."""
 
-    can_create = [SystemProcess()]
+    can_create = [SystemProcess(), DisableIfReadOnly()]
```

Comment:
could you describe what does it solve ? not sure I understand the use case.

Reply:
The primary use case for the read-only mode is to make it easier for migrating InvenioRDM instances to new VMs without losing write operations.
We already have a slightly simpler version of this hacked into our own TUW modules.

For example, for us it came in handy when we were migrating our instance from the test environment VM to the production VM:
We set the old instance in RO mode, cloned it to the new VM (DB & files), and started running the new instance (not in RO mode).
After that, we just needed to change the DNS entry to point to the new VM rather than the old one, and we were done.

Given that write operations were prohibited in RO mode, we are very certain that there were no lost writes during the migration.
Unfortunately, this is only true for write operations that go through a service layer with these PRs and doesn't cover e.g. user profile edits.